### PR TITLE
Fix some dark mode visibility issues

### PR DESCRIPTION
**Pull request by asgdf**
This changes this: 
grafik 
to this:
grafik
The color attribute is repeated in the :hover rule because otherwise `.dark .popupmenu button:hover` would overwrite it (with black). Not sure if there's a more elegant way to solve this.

It also changes this:
grafik
to this:
grafik

Lastly, this also changes subtle links to not be black in dark mode by inheriting the regular text color. Zarel mentioned fixing that in the dev room a while ago (and it is indeed fixed on the site), but apparently that change was never actually committed.

### CODE REVIEW

File: style/client.css [hunk 1]
```diff
@@ -178,17 +178,20 @@ button:disabled {
 }
 */
 .dark .button,
-.dark .tabbar a.button {
+.dark .tabbar a.button,
+.dark .popupmenu button.button {
```
Comment (Zarel) [inline style/client.css]: So, why is this necessary?

Comment (asgdf) [inline style/client.css]: Because otherwise the buttons look like in the top picture, based on the '.button, .popupmenu button.button' rule on line 330.

Comment (Zarel) [inline style/client.css]: Because `.popupmenu button` has more specificity than `.button`... ugh

Comment (Zarel) [inline style/client.css]: I'm unhappy with specificity rules :(

Comment (asgdf) [inline style/client.css]: I guess another class could be added to all the other popupmenu buttons, then the rules for them can be made more specific to not include these two buttons?

**Comment by asgdf** [comment]
Okay, now I'm not adding any rules that are just there for the specifity. In fact, now there are less of them than there were before.

**Comment by Zarel** [comment]
No, I think the old version was better...

**Comment by asgdf** [comment]
Aw, okay, I reverted it back to the old one. What made it worse though?

**Comment by Zarel** [comment]
I don't like the idea of adding a class to every menubutton ever.

**Comment by asgdf** [comment]
Is there a better way to do it than this one, then?

**Comment by Zarel** [comment]
I don't have any better ideas, sadly.
